src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -358,6 +361,14 @@ public override Symbol AssociatedSymbol
             }
         }
 
+        public bool LocalsAreZeroed
```

Comment:
I think this should be virtual on MethodSymbol and just return "true" in the default implementation. Then you do not need to type-check in MethodCompiler. 
You will need this anyways when dealing with synthesized members.

Reply:
Consider the name: AreLocalsZeroed. The rationale here is that we should try to prefix boolean member names with a true / false word: skip, is, are, etc ....

Reply:
Good suggestion.

Reply:
All occurrences of LocalsAreZeroed were changed to AreLocalsZeroed.